proguard-tests/build.gradle.kts

```diff
@@ -4,6 +4,14 @@ android {
     }
 
     buildTypes {
+        named("debug").configure {
```

Comment:
Why'd we add this back? It adds like 5+ mins to a full debug build. Shouldn't we only run it in CI?

Reply:
Oh damn I forgot why we removed it.  Android Studio gets pretty angry when not all modules have matching configurations and this made it happy.

Let me see if I can appease Android Studio without compromising time.

Reply:
Ohhh, yeah, I remember seeing those warnings. Let's use a variant filter then: https://developer.android.com/studio/build/build-variants#filter-variants

Reply:
Yep it looks like that's already working with `setIgnore`.  I pushed a commit to simplify the `debug` config to the minimum that makes AS happy.

I did a `time` of some clean builds and the extra config doesn't cost us anything.  Will confirm via Travis logs.

Reply:
Awesome, SGTM!

Reply:
Travis confirmed: no slowdown.